code/game/objects/structures/crates_lockers/crates/secure.dm

```diff
@@ -38,11 +38,16 @@
 	explosion(src, heavy_impact_range = 1, light_impact_range = 5, flash_range = 5)
 	qdel(src)
 
+/datum/armor/crate_secure/weapons // return to emitters
+	melee = 100
+	bullet = 100
+
 /obj/structure/closet/crate/secure/weapon
-	desc = "A secure weapons crate."
+	desc = "A secure weapons crate. The specially alloyed material hardens in response to physical trauma."
 	name = "weapons crate"
 	icon_state = "weaponcrate"
 	base_icon_state = "weaponcrate"
+	armor_type = /datum/armor/crate_secure/weapons
 
 /obj/structure/closet/crate/secure/plasma
```

Comment:
energy guns, laser carbines, flamethrowers come in these crates;
laser guns, stingbangs, batons come in /obj/structure/closet/crate/secure/gear
the only real guns that actually come in /obj/structure/closet/crate/secure/weapon are shotguns, battle rifles, and thermal pistols.

Reply:
oh you can also get a single laser gun to come in an unlocked using the cargo budget with an express console

Reply:
Thanks, I've filed a fix for that; Goody Packs weren't supposed to be accessible from the Cargo budget but the Express Console didn't consider this when building its' inventory list for purchase.

https://github.com/tgstation/tgstation/pull/88635